frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -301,7 +296,12 @@ export default function SurveyEdit(): JSX.Element {
                                             <div className="flex gap-4">
                                                 <PresentationTypeCard
                                                     active={value === SurveyType.Popover}
-                                                    onClick={() => onChange(SurveyType.Popover)}
+                                                    onClick={() => {
+                                                        onChange(SurveyType.Popover)
+                                                        if (survey.schedule === 'always') {
```

Comment:
if we don't have types yet, we should create one, hardcoding the same string many times over is just a new source of bugs

Reply:
like the `SurveySchedule`?

Reply:
> however, they all accomplish the same thing.

during execution yes, [but less room for bugs](https://en.wikipedia.org/wiki/Defensive_programming)